src/main/java/de/tum/in/www1/artemis/domain/participation/StudentParticipation.java

```diff
@@ -133,7 +133,7 @@ public boolean isTestRun() {
     }
 
     public void setTestRun(boolean testRun) {
-        this.testRun = testRun;
+        this.testRun = Boolean.TRUE.equals(testRun);
```

Comment:
I am a little bit confused by this. `boolean` can not be null

Reply:
yes, this is not really needed and should be simplified

Reply:
because I encountered a problem (before pulling the latest develop), Andi recommended me this approach. But i can revert this change.

Reply:
done in 870b79acbad5dc3c811008a691294b143b55668b